Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -629,7 +629,12 @@ public void Swap()
                 );
                 fork.Append(forkTip, DateTimeOffset.UtcNow, render: false);
 
+                string previousNamespace = _blockChain.Id.ToString();
                 _blockChain.Swap(fork);
+
+                Assert.Empty(_blockChain.Store.IterateIndex(previousNamespace));
+                Assert.Empty(_blockChain.Store.ListAddresses(previousNamespace));
```

Comment:
It would be better to check tx nonce.
```suggestion
                Assert.Empty(_blockChain.Store.ListAddresses(previousNamespace));
                Assert.Empty(_blockChain.Store.ListTxNonces(previousNamespace));
```